package.json

```diff
@@ -82,7 +82,7 @@
     "karma-jasmine": "^0.2.2",
     "karma-jasmine-matchers": "^0.1.3",
     "karma-phantomjs-shim": "^1.0.0",
-    "karma-phantomjs2-launcher": "^0.3.2",
+    "karma-phantomjs-launcher": "^1.0.1",
```

Comment:
I think this also requires a change to the `npm-shrinkwrap.json` file. The easiest way to do this is ensure you're on a version of npm greater then 3 (`nvm use` in the repo) and then `npm update --save karma-phantomjs-launcher@^1.0.1`. I *think* newer versions of npm will update the shrinkwrap.